sys/sys_amd64.const

```diff
@@ -56,7 +56,7 @@ DN_RENAME = 16
 EFD_CLOEXEC = 524288
 EFD_NONBLOCK = 2048
 EFD_SEMAPHORE = 1
-EPOLLET = 2147483648
+EPOLLET = 18446744071562067968
```

Comment:
Strange, I don't know how this happen. It seems that EPOLLET (1<<31) is extracted as 64bits value with a sign extension (0xffffffff80000000).

I need to found why before regenerating the `sys/sys_*.go` files because the value is changed in the *.go too (or correct it manually after the generation)

Here the versions I use :
- Ubuntu 16.04.1 LTS
- GCC system : gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.4)
- GCC to build linux kernel : gcc-7 (Ubuntu 7.1.0-6ubuntu2~16.04.york0) 7.1.0
- Go : go version go1.8.1 linux/amd64
- Linux kernel : 4.11.5

I also tried to extract linux constants with gcc-7 (make extract CC=...) but same behaviour for me.